appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -386,6 +386,38 @@ paths:
           description: Activity of this user.
           schema:
             $ref: '#/definitions/PagedDayActivityOverviews'
+  '/users/{userID}/activity/days/{date}/{goalID}':
```

Comment:
You earlier said you're not happy with the two consecutive IDs. What about defining the URL this way?
``/users/{userID}/activity/days/{date}/details/{goalID}``.
The additional benefit is that we can define other actions and things on a date too.